R/parallel.R

```diff
@@ -70,13 +72,15 @@ test_files_parallel <- function(
     load_package = load_package
   )
 
-  with_reporter(reporters$multi, {
-    parallel_updates <- reporter$capabilities$parallel_updates
-    if (parallel_updates) {
-      parallel_event_loop_smooth(queue, reporters, test_dir)
-    } else {
-      parallel_event_loop_chunky(queue, reporters, test_dir)
-    }
+  withr::with_dir(test_dir, {
```

Comment:
I had to do this so that `SnapshotReporter$end_reporter()` was called in the correct directory. Does that seem ok @gaborcsardi? If so, I can remove the `test_dir` args from the event loops.